### PR TITLE
Added Russian translation

### PR DESCRIPTION
**Pull request by shockpast**

### CODE REVIEW

**Comment by shockpast** [comment]
I didn't see where "Toggle Key" is being used, but also forgot to remove it from file.

**Comment by shockpast** [comment]
> I didn't see where "Toggle Key" is being used, but also forgot to remove it from file.

Is it something that like "Press `key` once to enable minimap, and then press again to disable it"?

**Comment by StyledStrike** [comment]
Oh the toggle key is used here:

tkey

It's meant to be a key you press to show/hide the minimap, and its not binded by default.

**Comment by shockpast** [comment]
> Oh the toggle key is used here:
> 
> tkey
> 
> It's meant to be a key you press to show/hide the minimap, and its not binded by default.

I wonder how to name it on russian, because all my variants in head are absolutely horrible, so i'll most likely just remove it, and leave it for another friend of mine, who knows english better. @Blueberryy

**Comment by StyledStrike** [comment]
I'll leave it there for now. When/if you or your friend find a suitable translation for the toggle key, feel free to open a pull request, and then I'll put it on the workshop. Thanks!

**Comment by Blueberryy** [comment]
> > Oh the toggle key is used here:
> > tkey
> > It's meant to be a key you press to show/hide the minimap, and its not binded by default.
> 
> I wonder how to name it on russian, because all my variants in head are absolutely horrible, so i'll most likely just remove it, and leave it for another friend of mine, who knows english better. @Blueberryy

I think it should be "Клавиша  переключения" or "Кл. переключения" if you want to fit window in more small resolution.
Also, i need to review few things on translation file.

File: resource/localization/ru/gminimap.properties [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+gminimap.border_thickness=Толщина обводки
+gminimap.border_color=Цвет обводки
+gminimap.terrain=Ландшафт
+gminimap.terrain_color=Цвет/Оттенок
+gminimap.terrain_brightness=Яркость
+gminimap.terrain_saturation=Насыщенность
+gminimap.terrain_color_inv=Инвертировать цвета
+gminimap.terrain_lighting=Показывать освещение
+gminimap.health_armor=Кастомное здоровье/броня
+gminimap.health_show_custom=Показывать кастомное здороьве/броню
```
Comment (Blueberryy) [inline resource/localization/ru/gminimap.properties:25]: опечатка в "здоровье"

**Comment by shockpast** [comment]
> I think it should be "Клавиша переключения" or "Кл. переключения" if you want to fit window in more small resolution.
Also, i need to review few things on translation file.

Файлик перевёл за 5 минут, чисто чтобы был (его не было изначально), благодарю за помощь <3

File: resource/localization/ru/gminimap.properties [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+gminimap.height=Высота
+gminimap.border_thickness=Толщина обводки
+gminimap.border_color=Цвет обводки
+gminimap.terrain=Ландшафт
+gminimap.terrain_color=Цвет/Оттенок
+gminimap.terrain_brightness=Яркость
+gminimap.terrain_saturation=Насыщенность
+gminimap.terrain_color_inv=Инвертировать цвета
+gminimap.terrain_lighting=Показывать освещение
+gminimap.health_armor=Кастомное здоровье/броня
```
Comment (Blueberryy) [inline resource/localization/ru/gminimap.properties:24]: Не знаю, подойдёт ли здесь "пользовательское", но можно заменить "Кастомное" на "Своё" или полностью заменить на "Здоровье/броня GMinimap".

File: resource/localization/ru/gminimap.properties [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+gminimap.health_show_custom=Показывать кастомное здороьве/броню
+gminimap.health_hide_default=Спрятать кастомное здоровье/броню
+gminimap.health_height=Высота полоски здоровья/брони
+gminimap.health_color=Цвет здоровья
+gminimap.low_health_color=Цвет низкого здоровья
+gminimap.armor_color=Цвет брони
+gminimap.landmarks=Точки
+gminimap.landmarks_help1=Правой кнопкой мыши, чтобы добавить точку на карту
+gminimap.landmarks_help2=Удерживайте левую кнопку мыши чтобы двигать карту, и колёсико для приближения
```
Comment (Blueberryy) [inline resource/localization/ru/gminimap.properties:33]: добавь запятую после "мыши" перед "чтобы".

File: resource/localization/ru/gminimap.properties [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+gminimap.scale=Масштаб
+gminimap.builtin_icons=GMinimap Icons
```
Comment (Blueberryy) [inline resource/localization/ru/gminimap.properties:35]: Можно перевести как "Значки GMinimap".

File: resource/localization/ru/gminimap.properties (hunk 2 above)
Comment (shockpast) [reply resource/localization/ru/gminimap.properties:24]: Как насчёт "польз." дабы влезать в мелкие разрешения экранов, читается довольно понятно.

Comment (Blueberryy) [reply resource/localization/ru/gminimap.properties:24]: можно

**Comment by Blueberryy** [comment]
> > I think it should be "Клавиша переключения" or "Кл. переключения" if you want to fit window in more small resolution.
> > Also, i need to review few things on translation file.
> 
> Файлик перевёл за 5 минут, чисто чтобы был (его не было изначально), благодарю за помощь <3

спасибо за перевод, пастыч 😄

**Pull request merged**